rca/home/models.py

```diff
@@ -309,9 +309,12 @@ def related_news_events_formatter(
         }
         meta = PAGE_META_MAPPING.get(page.__class__.__name__, "")
         description = PAGE_DESCRIPTION_MAPPING.get(page.__class__.__name__, "")
-
+        try:
+            image = get_listing_image(page).get_rendition("fill-878x472").url
+        except AttributeError:
+            image = None
         return {
-            "image": get_listing_image(page).get_rendition("fill-878x472").url,
+            "image": image,
```

Comment:
Something I noticed locally when running the home page with no images, thought I'd add it in